src/transform/ElementUtilities.js

```diff
@@ -1,4 +1,21 @@
 /**
+ * Polyfill function that tells whether a given element matches a selector.
+ * @param {!Element} el Element
+ * @param {!string} selector Selector to look for
+ * @returns {boolean} Whether the element matches the selector
+ */
+const matchesSelectorCompat = (el, selector) => {
```

Comment:
@mhurd, `compat` is an Android convention. Do you mind if I take an action item for this? I was hoping we could discuss polyfill strategy in general.